unstructured_ingest/v2/processes/connectors/sql/snowflake.py

```diff
@@ -124,7 +153,7 @@ class SnowflakeUploaderConfig(PostgresUploaderConfig):
 
 
 @dataclass
-class SnowflakeUploader(PostgresUploader):
+class SnowflakeUploader(SQLUploader):
     upload_config: SnowflakeUploaderConfig = field(default_factory=SnowflakeUploaderConfig)
```

Comment:
@rbiseck3 @potter-potter 
A lot of Snowflake classes inherit after `postgres` connector. I think it would be best to inherit from `sql` only.

Reply:
Agree. Let's change that for all the source inherited classes if possible.

Reply:
>  for all the source inherited classes

So only Indexer and Downloader (with config)?
If yes, it's changed

Reply:
As per Slack conversation, changed inheritance for every Snowflake class.